lib/modules/us_core_guidance/us_core_r4_medication_list_sequence.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Inferno
+  module Sequence
+    class USCoreR4MedicationListSequence < SequenceBase
+      group 'US Core R4 Profile Conformance'
+
+      title 'Medication List Guidance Tests'
+
+      description 'Verify that MedicationRequests can be queried according to the US Core R4 Medication List Guidance'
+
+      test_id_prefix 'MLG'
+
+      requires :token, :patient_id
+      conformance_supports :MedicationRequest, :Medication
+
+      details %(
+        The #{title} Sequence tests that MedicationRequests and Medications can
+        be retrieved according to the [US Core R4 Medication List
+        Guidance](https://www.hl7.org/fhir/us/core/all-meds.html)
+      )
+
+      test :include_medications do
+        metadata do
+          id '01'
+          name 'FHIR server supports including Medications in a MedicationRequest search'
+          link 'https://www.hl7.org/fhir/us/core/all-meds.html#options-for-representing-medication'
+          description %(
+            When referencing the Medication resource, the resource may be...an
```

Comment:
why using ... here?

Reply:
I'm quoting from the link but leaving out the parts about the other options for referencing Medications:
> When referencing the Medication resource, the resource may be included in the bundle returned, an external resource, or a contained if the resource can’t stand alone. These options are shown in figure 3 below. The server application MAY choose any combination of these methods, but if an external reference to Medication is used, the server SHALL support the include parameter for searching this element.